api/src/tests/transactions_test.rs

```diff
@@ -1387,19 +1387,15 @@ async fn simulation_failure_error_message() {
     //     }
     // }
 
-    let output = context.simulate_transaction(&admin0, json!({
+    let resp = context.simulate_transaction(&admin0, json!({
         "type": "script_payload",
         "code": {
             "bytecode": "a11ceb0b030000000105000100000000050601000000000000000600000000000000001a0102",
         },
         "type_arguments": [],
         "arguments": [],
-    }), 200).await;
-
-    assert!(output.as_array().unwrap()[0]["vm_status"]
-        .as_str()
-        .unwrap()
-        .contains("Division by zero"));
```

Comment:
is this contains still checked?

Reply:
I see golden again

Reply:
Hmm where in the golden file? Didn't see this message.

Reply:
Copy paste gone wrong, updated, thanks for catching!